config/webhook.yaml

```diff
@@ -40,10 +40,10 @@ spec:
       serviceAccountName: webhook
       containers:
       - name: webhook
+        imagePullPolicy: Always
```

Comment:
This is more or less pointless with `ko` since you'll get a digest in the image fields.